noxfile.py

```diff
@@ -26,11 +26,12 @@
 BLACK_VERSION = "black==19.3b0"
 BLACK_PATHS = ["docs", "google", "tests", "noxfile.py", "setup.py"]
 
-if os.path.exists("samples"):
-    BLACK_PATHS.append("samples")
+DEFAULT_PYTHON_VERSION = "3.7"
+SYSTEM_TEST_PYTHON_VERSIONS = ["2.7", "3.7"]
+UNIT_TEST_PYTHON_VERSIONS = ["2.7", "3.5", "3.6", "3.7", "3.8"]
```

Comment:
This will be overridden on the next autogen pass, shouldn't we add the rules to `synth.py` to fill in the blanks? Proposing because this is what actually gets generated:
```py
DEFAULT_PYTHON_VERSION = ""
SYSTEM_TEST_PYTHON_VERSIONS = []
UNIT_TEST_PYTHON_VERSIONS = []
```